.golangci.yml

```diff
@@ -6,6 +6,7 @@ linters:
     - bodyclose
     - containedctx
     - contextcheck
+    - copyloopvar
```

Comment:
Yeah, just doing some proactive maintenance here. The code changes here are safe for Go 1.21 since there's no `t.Parallel` in play... and with Go 1.22 that whole pattern will thankfully be a thing of the past. The linters are both smart enough to remain dormant until the module adopts Go 1.22.